docs/components/rating/README.md

```diff
@@ -188,16 +188,19 @@ Change size for the star icon and text. Default size is medium.
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines:
+Many WCAG requierments are contextual to their implementation. 
+To ensure that usage of this component complies with accessibility guidelines you are responcible for the following:
 
-- Use Rating component only on light backgrounds
+- Always provide a `href` attribute. Empty `href` attributes are not considered true links and are not accessible to keyboard users.
```

Comment:
? href is not required in our rating component, is this just copy pasta from the links section or should we fix that in rating?

Reply:
this requirement can be articulated better - "when linking the rating component be sure to provide a valid href value" or something